internal/sveltos/status.go

```diff
@@ -66,6 +68,30 @@ func GetStatusConditions(summary *sveltosv1beta1.ClusterSummary) ([]metav1.Condi
 	return conditions, nil
 }
 
+// CreateEventFromCondition creates sveltos related kubernetes event from the provided condition.
+func CreateEventFromCondition(object runtime.Object, generation int64, targetCluster client.ObjectKey, condition *metav1.Condition) {
```

Comment:
The events created with this function are the ones that show up in the controller logs but not with kubectl, as explained in the PR desciption.

Reply:
Resolving this comment to proceed with merging. This problem relates to all of the events and can be addressed separately.